### DISCUSSION TITLE
Objects-page not building

### DISCUSSION THREAD

**Discussion by salger1** [thegetty/quire#939] [Get Help]
Hello! I have just started to incorporate the filterable object grid to an existing Quire project (https://nelson-atkins.org/starr1/) and I am running into an issue when I go to preview the project: 

`[11ty] You’re trying to use a layout that does not exist: objects-page (undefined) (via Error)
2024-07-18T19:05:53.253Z Eleventy:EleventyErrorHandler (error stack): Error: You’re trying to use a layout that does not exist: objects-page (undefined)
`

Any ideas why it doesn't recognize the page layout/how to fix/update our project? 

`'@thegetty/quire-11ty': '1.0.0-rc.16',
  npm: '9.6.7',
  node: '18.17.1'`

**Comment by Erin-Cecele** [comment]
@salger1 is there a branch I can look at / test on?

**Reply by salger1** [reply]
Yes! The combined branch (which now has all 4 volumes and still runs, thank you again for that help). I just pushed up the testing I was doing so you can grab it.

**Reply by salger1** [reply]
Ok so here is an update: I talked to Rachel and she thinks we never fully updated to rc.16 and that most of the files are rc.10. I built a test 16 version and it has the layout file for objects-page. Do I have to fully update the catalogue to get this to work or can I pull specific files?

**Comment by Erin-Cecele** [comment]
Hi @salger1, Ah hah, yes, that would explain it. @geealbers and I will need to look into whether there is an easy way to incorporate the new feature without having to merge your existing project with an entirely new one. Stay tuned. 

Another thing I want to point out about this feature (and something I need to update in the documentation) is that this filter only operates at a subfolder level. So, in the case of your project, that would require moving Vol. 1-4 into a folder and then adding an index.md file with the `layout: objects-page`. That is the way this feature currently works, but it's something that we can consider altering in the future.

**Comment by Erin-Cecele** [comment]
Hi @salger1, I just wanted to touch base and see where things left off with this. Did you still want to incorporate the `objects-page` into your project?

**Reply by salger1** [reply]
Sorry Erin! I think we decided not to bother updating to the newest version of Quire right now. I'll let you know if we change our minds in the future.

**Reply by Erin-Cecele** [reply]
Understood. Thanks for the update @salger1
